transform/mattermost-analytics/models/staging/stripe/stg_stripe__products.sql

```diff
@@ -0,0 +1,39 @@
+with source as (
+
+    select * from {{ source('stripe_raw', 'products') }}
+
+),
+
+products as (
+
+    select
+        active,
+        attributes,
+        created,
```

Comment:
Timestamps should follow the postfix `_at`. 
```suggestion
        created AS created_at,
```